gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -133,9 +133,6 @@ private static void configureErrorProneOptions(
                 "CanIgnoreReturnValueSuggester",
                 "InlineMeSuggester",
                 "PreferImmutableStreamExCollections",
-                // StringCaseLocaleUsage duplicates our existing DefaultLocale check which is already
-                // enforced in some places.
-                "StringCaseLocaleUsage",
```

Comment:
This effectively changes this check from `SUGGESTION` to `WARNING`, but we've had an excavator for this for a while now, so this should be acceptable.